releasenotes/notes/apm-status-code-metrics-95766efbc01100e1.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM: The "http.status_code" tag is now supported as a numeric value too when computing APM trace stats.
```

Comment:
Should it be noted that the numeric value will take precedence if values appear in both Metrics and Meta?

Reply:
I've made that change. It's generally confusing to a user and doesn't make much sense that you can have two tags with the same key but different value types. I added it anyway because there may be situations like:
```go
span.SetTag("http.status_code", 200)
// ... stuff
span.SetTag("http.status_code", "302")
```
And then 302 is ignored, and that would be confusing. BUT the above scenario is frankly ridiculous. I'm ok with being thorough though!

Reply:
Wouldn't this be a breaking change if the code above used to result in `302` before, but now it's `200` in 7.39.0+? 

>  the above scenario is frankly ridiculous

Agreed, but avoiding the (improbable but not impossible) breaking change should be easy: give `meta` higher priority and look at `metrics` only as a fallback.

Reply:
Also, the frontend/backend give `http.status_code` special significance, but I don't know if they look only in `meta` like the trace agent does currently (before this PR).